src/extensions/renderer/canvas/drawing-edges.js

```diff
@@ -181,45 +181,42 @@ CRp.drawArrowheads = function( context, edge, drawOverlayInstead ){
 CRp.drawArrowhead = function( context, edge, prefix, x, y, angle ){
   if( isNaN( x ) || x == null || isNaN( y ) || y == null || isNaN( angle ) || angle == null ){ return; }
 
-  var self = this;
   var arrowShape = edge.pstyle( prefix + '-arrow-shape' ).value;
-
-  if( arrowShape === 'none' ){
-    return;
-  }
-
-  var gco = context.globalCompositeOperation;
+  if( arrowShape === 'none' ) { return; }
 
   var arrowClearFill = edge.pstyle( prefix + '-arrow-fill' ).value === 'hollow' ? 'both' : 'filled';
   var arrowFill = edge.pstyle( prefix + '-arrow-fill' ).value;
+  var arrowWidth = edge.pstyle( 'width' ).pfValue;
```

Comment:
Should not be called `arrowWidth`.  That's actually a different value, semantically.  Better to call it `edgeWidth` or it will be confusing.